server/base/src/main/java/org/apache/accumulo/server/util/VerifyTabletAssignments.java

```diff
@@ -95,7 +97,10 @@ private static void checkTable(final ClientContext context, final boolean verbos
 
     final HashSet<KeyExtent> failures = new HashSet<>();
 
-    Map<HostAndPort,List<KeyExtent>> extentsPerServer = new TreeMap<>();
+    Comparator<HostAndPort> comparator = Comparator.nullsFirst(
```

Comment:
You should be able to make this static and create it just one time in the class and reuse the comparator just like we were doing originally in the copied version. It might be better to even have it in a public utility class somewhere so we could re-use it elsewhere if needed.

Reply:
Addressed in 6495b62